contracts/src/ModuleCRC20.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.6.11;
+
+import "ds-token/token.sol";
+
+contract ModuleCRC20 is DSToken  {
```

Comment:
what is DSToken?

Reply:
https://github.com/dapphub/ds-token/blob/master/src/token.sol
A base implementation of erc20 token.